post-processor/checksum/post-processor.go

```diff
@@ -24,7 +24,6 @@ import (
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
 
-	Keep          bool     `mapstructure:"keep_input_artifact"`
```

Comment:
this is not being used so I decided to remove but I'm not sure if this is a good idea

Reply:
this config option is used elsewhere in the core. I suspect someone added it here because they didn't understand how it was supposed to work, so I think it's fine to remove.